gui/GameConqueror.py

```diff
@@ -1040,9 +1039,26 @@ def check_backend_version(self):
         if self.backend.get_version() != VERSION:
             self.show_error(_('Version of scanmem mismatched, you may encounter problems. Please make sure you are using the same version of GameConqueror as scanmem.'))
 
-    
 
 if __name__ == '__main__':
+    # Parse cmdline arguments
+    parser = argparse.ArgumentParser(prog='GameConqueror',
+                                     description="A GUI for scanmem, a game hacking tool"
+
+                                    )
+    parser.add_argument('-v', '--version', action='version', version='%(prog)s ' + VERSION)
+    parser.add_argument("pid", nargs='?', type=int, help="PID of the process")
+    args = parser.parse_args()
+
+    # Init application
     GObject.threads_init()
     Gdk.threads_init()
-    GameConqueror().main()
+    gc_instance = GameConqueror()
+
+    # Attach to given pid (if any)
+    if (args.pid is not None) :
+        process_name = os.popen('ps -p ' + str(args.pid) + ' -o command=').read().strip()
+        gc_instance.select_process(args.pid, process_name)
+    
```

Comment:
You mean that you want all your empty lines with no starting spaces?
The whole file is inconsistent on this, I thought it didn't matter.
A later commit actually fixes it.